app/scripts/modules/amazon/src/serverGroup/configure/AmazonImageSelectInput.tsx

```diff
@@ -348,6 +348,7 @@ export class AmazonImageSelectInput extends React.Component<IAmazonImageSelector
 
       return (
         <div className="col-md-9">
+          {/* @ts-ignore */}
```

Comment:
I didn't have a whole lot of context on why this [customization](https://github.com/spinnaker/deck/blob/master/app/scripts/modules/core/src/presentation/TetheredSelect.tsx#L6) was needed in the first place. So making `ts` ignore the failure.

Reply:
what was the failure?

Reply:
```
Type '{ menuRenderer: (params: MenuRendererProps<OptionValues>) => HandlerRendererResult; isLoading: boolean; placeholder: string; ... 73 more ...; ref?: LegacyRef<...>; }' is not assignable to type 'IntrinsicClassAttributes<TetheredSelect>'.
  Types of property 'ref' are incompatible.
    Type 'LegacyRef<ReactSelectClass<any>>' is not assignable to type 'LegacyRef<TetheredSelect>'.
      Type 'RefObject<ReactSelectClass<any>>' is not assignable to type 'LegacyRef<TetheredSelect>'.
        Type 'RefObject<ReactSelectClass<any>>' is not assignable to type 'RefObject<TetheredSelect>'.
          Property '_renderOuter' is missing in type 'ReactSelectClass<any>' but required in type 'TetheredSelect'.
```